cli/src/wallet_cli/mod.rs

```diff
@@ -79,6 +79,8 @@ pub enum WalletCommand {
         fixed_cost: u64,
         /// The staking period (in epochs). Will default to the staking unbonding period.
         staking_period: Option<u32>,
+        /// Whether to allot mana from the account output.
+        allot_from_account: bool,
```

Comment:
This is required? Sounds annoying, would default to true so one doesn't has to provide it every time

Reply:
good point. Should I default to true or false though 🤔

Reply:
I would default to true, what's your opinion on this @thibault-martinez since you opened the issue :trollface: